### DISCUSSION TITLE
422 (Unprocessable Content) Bug when uploading more than 2mb files in production (live server)

### DISCUSSION THREAD

**Discussion by AHMED-GAMAL-AG** [filamentphp/filament#11565] [General]
the upload functionality works fine in my local environment, but on production, i get 422 

![image](https://github.com/filamentphp/filament/assets/76778937/53fde035-f675-4ece-876a-4e5ad890063b)

here is the live wire configuration and the  ini.php file

![image](https://github.com/filamentphp/filament/assets/76778937/75d29fa5-d222-405c-b3cb-30616d0717e4)
![image](https://github.com/filamentphp/filament/assets/76778937/e2a22e41-037a-437b-9b3a-91fc1485cfe0)

**Comment by dissto** [comment]
Assuming you are using nginx. Perhaps take a look at [this](https://serverfault.com/a/1003458)
